performance-es.md

```diff
@@ -57,3 +57,117 @@ La optimización prematura también puede afectarte al atarte a ciertas decision
 En la gran mayoría de casos, el tamaño y velocidad del programa no es el problema. La optimización más fácil es no hacerla. La segunda alternativa más fácil es simplemente comprar mejor hardware.
 
 Cuando hayas decidido cambiar tu programa, sigue leyendo.
+
+### Modificar los datos
+
+Modificar los datos significa agregar o alterar la representación de los datos
+que estas procesando. Desde el punto de vista del rendimiento, algunos de
+estos acabaran cambiando la complejidad O() asociada a diferentes aspectos de
+la estructura de datos.
+
+Ideas para mejorar tu estructura de datos:
+
+* Campos adicionales
+
+  El clásico ejemplo de esto es almacenar la longitud de una lista enlazada en un
+  campo en el nodo raíz. Mantenerla actualizada lleva un poco más de trabajo,
+  pero consultar la longitud se convierte en un simple acceso a un campo en vez de una
+  búsqueda transversal con complejidad O(n). Tu estructura de datos puede presentar una
+  mejora similar: un poco de mantenimiento en algunas operaciones a cambio de
+  mejorar el rendimiento en un caso de uso común.
+
+  De manera similar, almacenar punteros a nodos frecuentemente utilizados en vez
+  de realizar búsquedas adicionales. Esto cubre cosas como el link "hacia atrás"
+  en una lista doblemente enlazada para hacer que la eliminación de nodos tenga una complejidad O(1).
+  Algunas listas de salto guardan un "puntero de búsqueda", donde almacenan un
```

Comment:
`s/almacenan/almacenas` or `s/almacenan/se almacena`

Reply:
This one I'm not convinced about. I'm not 100% certain but I think it's up to the implementation of the list to keep the pointer, so `almacenas` sounds off to me. Given that you don't really have to do anything, it's the list that does that. I may be confused though, I'm not totally familiarized with skip lists to be honest.

Reply:
Like you, I am not too familiar with skips lists, however, I'm basing my suggestion on the way the English text is written and of the tone of the text itself. It appears to me that this text is directed to the reader and his/her actions and code, since the English text continuously uses `you` and `your`. For that reason, I still think `almacenas` or perhaps `puedes almacenar` might a better translation.

Using the English text as reference: `Some skips lists keep a "search finger", where you store a pointer...`. The text talks about what the reader can do in this `search finger` location. I think the Spanish text should state the same. Let me know what you think.

Reply:
Actually, you are correct. `Almacenas` is closer to the original text